src/Network/Network/Network.format.ps1xml

```diff
@@ -2526,11 +2526,11 @@
                 <PropertyName>IpAddress</PropertyName>
               </ListItem>
               <ListItem>
-                <Label>NetworkInterfaceIpConfiguration</Label>
+                <Label>NetworkInterfaceIpConfigurationId</Label>
                 <PropertyName>NetworkInterfaceIpConfigurationText</PropertyName>
```

Comment:
shouldn't we change this to NetworkInterfaceIpConfigurationIdText?

Reply:
No, that label is what is printed on the powershell console. I think it would read kind of weird to add 'text' to the output console

Reply:
Same as the other comment.